api/api.go

```diff
@@ -298,7 +305,7 @@ func (api *Server) GetChainMeta(ctx context.Context, in *iotexapi.GetChainMetaRe
 	timeDiff := (t2.Sub(t1) + 10*time.Second) / time.Millisecond
 	tps := float32(numActions*1000) / float32(timeDiff)
 
-	chainMeta := &iotextypes.ChainMeta{
+	chainMeta = &iotextypes.ChainMeta{
```

Comment:
no need to define a new var again
```go
chainMeta.NumAction = int64(totalActions)
chainMeta.Tps =
chainMeta.TpsFloat =
```

Reply:
also, i released protobuf 0.4.7 https://github.com/iotexproject/iotex-proto/releases/tag/v0.4.7
please update go.mod